src/app/features/public-form/edit-public-form-related-entities/edit-public-form-related-entities.component.ts

```diff
@@ -67,6 +67,13 @@ export class EditPublicFormRelatedEntitiesComponent
   }
 
   private initializeLinkedEntity(): void {
+    this.formControl.statusChanges.subscribe(() => {
+      const idControl = this.form.get("id");
```

Comment:
sorry, one more thought here :see_no_evil: 

Do we really need a formGroup or can we just simply create and use a signle formControl here (simplifying things like this line)?